docs/spelling_wordlist.txt

```diff
@@ -160,3 +160,4 @@ LinkType
 GroupType
 PathLike
 ElementType
+splitted
```

Comment:
don't add this

Reply:
It's because the headers under this would be too redundant on the left bar if we keep the original header levels. https://asyml-forte--730.org.readthedocs.build/en/730/